walletsetup.go

```diff
@@ -40,6 +41,64 @@ func networkDir(dataDir string, chainParams *chaincfg.Params) string {
 	return filepath.Join(dataDir, netname)
 }
 
+// displaySimnetMiningAddrs shows simnet mining addresses for the passed seed.
+// If imported is false, then only the SLIP0044 address is shown (because, by
+// default, the wallet is upgraded to the SLIP0044 coin type).
+func displaySimnetMiningAddrs(seed []byte, imported bool) error {
+	params := chaincfg.SimNetParams()
+	ctLegacyKeyPriv, ctSLIP0044KeyPriv, acctKeyLegacyPriv, acctKeySLIP0044Priv, err := udb.HDKeysFromSeed(seed, params)
+	if err != nil {
+		return err
+	}
+	ctLegacyKeyPriv.Zero()
+	ctSLIP0044KeyPriv.Zero()
+	defer acctKeyLegacyPriv.Zero()
+	defer acctKeySLIP0044Priv.Zero()
+
+	keys := map[string]*hdkeychain.ExtendedKey{
+		"legacy":   acctKeyLegacyPriv,
+		"SLIP0044": acctKeySLIP0044Priv,
+	}
+
+	// If imported is false, then the wallet was created with a new random
+	// seed and is automatically upgraded to the SLIP0044 coin type,
+	// therefore the legacy key is not applicable.
+	if !imported {
+		delete(keys, "legacy")
+	}
+
+	fmt.Println("")
+	fmt.Println("NOTE: only start the wallet after at least 2 blocks (i.e. blocks at heights 1")
+	fmt.Println("and 2) have been mined in the backing dcrd node, otherwise account and address")
+	fmt.Println("discovery may not work correctly.")
+	fmt.Println("")
+
+	for ct, acctKeyPriv := range keys {
```

Comment:
we might want to guarantee the order these are printed.  maybe:

```
for _, ct := range []string{"SLIP0044", "legacy"} {
    acctKeyPriv := keys[ct]
    ...
}
```

Reply:
Updated.